File: spec/code_samples/Python_requests/list@check_email@{email}/get.py
import requests

url = "https://analyzemail.com/api/v1/list/check_email/%7Bemail%7D"

querystring = {"username":"SOME_STRING_VALUE"}

headers = {'content-type': 'application/json'}

response = requests.request("GET", url, headers=headers, params=querystring)

print(response.text)

File: spec/code_samples/Php_http2/list@check_email@{email}/get.php
<?php

$client = new http\Client;
$request = new http\Client\Request;

$request->setRequestUrl('https://analyzemail.com/api/v1/list/check_email/%7Bemail%7D');
$request->setRequestMethod('GET');
$request->setQuery(new http\QueryString(array(
  'username' => 'SOME_STRING_VALUE'
)));

$request->setHeaders(array(
  'content-type' => 'application/json'
));

$client->enqueue($request)->send();
$response = $client->getResponse();

echo $response->getBody();